bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -136,6 +144,29 @@ <h2 class="go">{{ _('Download Firefox') }}</h2>
             <a class="fxos-link" href="{{ url('firefox.desktop.index') }}">{{ _('Learn more') }}</a>
           </div>
         </li>
+    {% if waffle.switch('mozorg-home-privacy-promo') and l10n_has_tag('privacy_day') %}
+      {% if waffle.switch('mozorg-home-privacy-promo-jan28') and LANG.startswith('en') %}
```

Comment:
Same micro-optimization nits as above: move the waffle.switch checks after the l10n_has_tag and LANG.startswith checks.